### ISSUE TITLE
Compatibility with community-dev

### ISSUE THREAD

**Issue by buttash** [akeneo-labs/EnhancedConnectorBundle#49]
Hi
Is it possible to use the bundle with community-dev?  I have asked this question on the Akeneo forums but not had any reply.

[http://www.akeneo.com/forums/topic/creating-reference-data-in-pim-community-standard/#post-6720](url)

Thanks in advance
Martin

**Comment by fitn** [comment]
Hi @buttash,

We're not compatible yet with the CE/EE dev-master branch.
I dont know when we will migrate (at the end of the month with the stable version I think).
You're free to help us on this task. We will take care of your PR and will release an alpha or beta version sooner in this case :)

Regards

**Comment by buttash** [comment]
Thanks for getting back.  I'm using the dev fork as I couldn't get reference data working on the community edition and didn't get any response on the main forum despite numerous posts on the subject.

It seems that the csv exports on dev do not support reference data options in the attribute options export plus the family export adds some more fields.  The latter is not compatible with PIMGento (PimGento will need changing).  The lack of reference data in the attribute option export means that I have to add the options manually.

I'd be happy to help test the enhanced connector on dev, plus (given some help) work on how to update the options exports to include reference data.  The family issue is something for me to take up with AgenceDnD.

Martin

**Comment by fitn** [comment]
Hi @buttash 

I just merged #51 to make this bundle compatible with the dev-master of Akeneo PIM. I compared CSV files and it was good on my dataset.
Feel free to test it with PimGento with your own dataset.

About reference datas, they are exported like options today. It needs custom export on PIM side  for each reference data as we can't know the fields and format needed.
I don't know pretty well PimGento but It seems you're right there is a missing feature: https://github.com/Agence-DnD/PIMGento/blob/master/app/code/community/Pimgento/Attribute/Helper/Data.php#L16

Regards,
